### PR TITLE
Add tooling tests for `persist init` command

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
$subject

### CODE REVIEW

File: tooling-tests/src/test/resources/test-src/samples/tool_test_init_2/Config.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+foo1 = "bar1"
```
Comment (MadhukaHarith92) [inline tooling-tests/src/test/resources/test-src/samples/tool_test_init_2/Config.toml:1]:
```suggestion
foo1 = "bar1"

```

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+oo1 = "bar1"
```
Comment (MadhukaHarith92) [inline tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml:1]: `tool_test_init_4` isn't there. Let's rename `tool_test_init_5` -> `tool_test_init_4` and the subsequent test sources.

File: tooling-tests/src/test/resources/test-src/samples/tool_test_init_5/Config.toml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+oo1 = "bar1"
```
Comment (MadhukaHarith92) [inline tooling-tests/src/test/resources/test-src/samples/tool_test_init_5/Config.toml:1]:
```suggestion
foo1 = "bar1"
```

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml (hunk 2 above)
Comment (MadhukaHarith92) [inline tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml:1]:
```suggestion
foo1 = "bar1"
```

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+user = "root"
+database = ""
+password = ""
+
+[foo]
+host = "wso2.com"
+port = 6688
+user = "john"
+password = "john123"
+database = "mariadb"
```
Comment (MadhukaHarith92) [inline tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml:12]:
```suggestion
[foo]
host = "wso2.com"
port = 6688
user = "john"
password = "john123"
database = "mariadb"

host = "localhost"
port = 3306
user = "root"
database = ""
password = ""
```

Comment (MadhukaHarith92) [reply tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml:12]: Let's update the expected output as above and disable this test case for now. Re-enable after the issue is sorted.

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml (hunk 2 above)
Comment (sahanHe) [reply tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml:1]: tool_test_init_4 is uses to test the outside ballerina project test case

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml (hunk 4 above)
Comment (sahanHe) [reply tooling-tests/src/test/resources/test-src/reference/tool_test_init_7/Config.toml]: sorted in [3cb66d3](https://github.com/ballerina-platform/persist-tools/pull/7/commits/3cb66d3d9ba065943e9b6d1b8df81da6d6f56135)

File: tooling-tests/src/test/resources/test-src/samples/tool_test_init_2/Config.toml (hunk 1 above)
Comment (sahanHe) [reply tooling-tests/src/test/resources/test-src/samples/tool_test_init_2/Config.toml]: Updated in [24eda25](https://github.com/ballerina-platform/persist-tools/pull/7/commits/24eda2553c3805131632ee9181c97b4242370b18)

File: tooling-tests/src/test/resources/test-src/samples/tool_test_init_5/Config.toml (hunk 3 above)
Comment (sahanHe) [reply tooling-tests/src/test/resources/test-src/samples/tool_test_init_5/Config.toml]: Updated in [24eda25](https://github.com/ballerina-platform/persist-tools/pull/7/commits/24eda2553c3805131632ee9181c97b4242370b18)

File: tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml (hunk 2 above)
Comment (sahanHe) [reply tooling-tests/src/test/resources/test-src/reference/tool_test_init_5/Config.toml]: Updated in [24eda25](https://github.com/ballerina-platform/persist-tools/pull/7/commits/24eda2553c3805131632ee9181c97b4242370b18)

**Review by MadhukaHarith92 — APPROVED** [review]

**Pull request merged**